examples/evaluation/huggingface-evaluation.ipynb

```diff
@@ -0,0 +1,1340 @@
+     "nuid": "cc4119e2-3f6c-4c28-b4a3-1b556fbeb547",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from transformers import pipeline\n",
+    "\n",
+    "summarizer = pipeline(\"summarization\", model=\"Falconsai/text_summarization\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "1a25d100-1d18-4cfa-a6b4-29466070651c",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Log model to mlflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "3e614ce7-4765-441a-bd87-34de7aa3787c",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/bl/thhgmbz97z9gt3wbq1d50_b80000gp/T/ipykernel_62961/199251651.py:4: FutureWarning: The 'transformers' MLflow Models integration is known to be compatible with the following package version ranges: ``4.25.1`` -  ``4.34.1``. MLflow Models integrations with transformers may not succeed when used with package versions outside of this range.\n",
+      "  model_info = mlflow.transformers.log_model(\n",
+      "/Users/ann.zhang/.pyenv/versions/3.8.13/lib/python3.8/site-packages/mlflow/models/model.py:619: FutureWarning: The 'transformers' MLflow Models integration is known to be compatible with the following package version ranges: ``4.25.1`` -  ``4.34.1``. MLflow Models integrations with transformers may not succeed when used with package versions outside of this range.\n",
+      "  flavor.save_model(path=local_path, mlflow_model=mlflow_model, **kwargs)\n",
+      "Your max_length is set to 200, but your input_length is only 10. Since this is a summarization task, where outputs shorter than the input are typically wanted, you might consider decreasing max_length manually, e.g. summarizer('...', max_length=5)\n",
+      "Registered model 'falconsai-summarization' already exists. Creating a new version of this model...\n",
+      "2023/12/01 15:23:27 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: falconsai-summarization, version 10\n",
+      "Created version '10' of model 'falconsai-summarization'.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import mlflow\n",
+    "\n",
+    "with mlflow.start_run():\n",
+    "    model_info = mlflow.transformers.log_model(\n",
+    "        transformers_model=summarizer,\n",
+    "        artifact_path=\"falcons\",\n",
+    "        input_example=\"Please summarize the following article:\\n article\",\n",
```

Comment:
Can we add the explicit signature inference usage and registration as well? (it's automatic, but for teaching purposes it can give the users exposure to inference parameters)